packages/next-swc/crates/core/tests/full.rs

```diff
@@ -81,7 +81,7 @@ fn test(input: &Path, minify: bool) {
                 auto_modularize_imports: None,
                 optimize_barrel_exports: None,
                 optimize_server_react: None,
-                disable_checks: false,
+                bundle_target: String::from("default").into(),
```

Comment:
This should be an enum, shouldn't it?

Reply:
yes, enum will be better, add a reminder for myself to refactor it later. thanks for suggestion 👍